src/intl/index.js

```diff
@@ -143,7 +143,7 @@ const getUrl = ({ getState, langPackLocale, componentKey }) => {
   const localeFilename = state.getIn(['config', 'localeFilename']);
 
   return `${[
-    moduleBaseUrl,
```

Comment:
can you include a test to cover this

Reply:
This may be easiest to test if you promote the `replace` to its own function.

Reply:
Good call @mtomcal I'm going to try that.

Reply:
updated with tests 6bc1c5f